fast_reliable.py

```diff
@@ -0,0 +1,62 @@
+import time
+from typing import Iterable
+
+
+def build_words_list_from_file_words(file_name: str) -> list:
+    """
+    :param file_name: File_text to read words.
+    :return: List with all words as values.
+    """
+    with open(file_name, 'r') as fdr:
```

Comment:
What is fdr? Franklin delano roosevelt? I think we should pick a different name (not informative enough)